sovware/directorist
# Icon support added for legacy templates

How to test:

1. Copy directorist v7.3.3 templates in your theme.
2. Enable "Legacy Icon Support" from settings.
3. All icons should be displayed in front-end.
